doc/comandi/converti-opzioni.md

```diff
@@ -1,54 +1,36 @@
-svantaggio è che il processo potrebbe introdurre errori di sintassi
-rST
+### `link-normattiva`
+
+Questa opzione, trasforma tutti i riferimenti a leggi e normative in collegamenti ipertestuali al sito http://www.normattiva.it/
 
 ### `celle-complesse`
 
-Se il vostro documento include celle di tabelle con molto testo ed una
-formattazione specifica, questo potrebbe causare errori di sintassi
-rST. Questo è dovuto al fatto che il contenuto di una cella viene
-portato a capo. Per evitare questo problema, `converti` evita di
-spezzare le righe quando selezionate l'opzione `celle-complesse`. Così
-facendo potete ottenere un documento senza errori di sintassi, ma con
-righe molto lunghe
+Questa opzione agisce sul contenuto delle celle, di default il contenuto di una cella viene portato a capo e spesso riporta errore nella sintassi rST. 
+Attivando l'opzione celle-complesse il processo di conversione evita di mandare a capo il contenuto delle celle riportando tutto il contenuto della cella sulla stessa riga.
 
 ### `preserva-citazioni`
 
-Molti utenti di Microsoft Word indentano il testo quando vogliono
-formattare una citazione. Di conseguenza, pandoc interpreta il testo
-indentato come citazione. Spesso non è quello che vogliamo, quindi
-`converti` rimuove tutte le citazioni dai documenti. Se volete
-preservare le citazioni potete farlo usando questa opzione
+Molti utenti di Microsoft Word indentano il testo quando vogliono formattare una citazione. Pandoc interpreta il testo indentato come citazione mentre 'converti' rimuove tutte le citazioni dai documenti. 
+Per preservare le citazioni è possibile usare l'opzione 'preserva-citazioni'
 
 ### `dividi-sezioni`
 
-Selezionando questa opzione chiedete a `converti` di spezzare il file
-`.rst` convertito in base ai capitoli, che verranno raccolti in una
-cartella `index/` e elencati in un file `index.rst`. Questo processo
-potrebbe causare errori di sintassi rST
+Questa opzione divide i capitoli in file .rst, li raccoglie in una cartella index/ e li elenca nel file index.rst.
 
 ## Come usare le opzioni
 
-Se siete arrivati a questa pagina dal convertitore web, potete
-selezionare queste opzioni usando la checkbox corrispondente
-nell'interfaccia web del convertitore.
+Se siete giunti a questa pagina dal convertitore web, è possibile selezionare queste opzioni usando la checkbox corrispondente, nell'interfaccia web del convertitore.
 
-Se state usando `converti` da riga di comando, queste opzioni possono
-essere indicate lanciando `converti --dividi-sezioni documento.odt`,
-per esempio.
+Se state usando `converti` da riga di comando, queste opzioni possono essere indicate lanciando `converti --dividi-sezioni documento.odt`
 
-Se state sviluppando un servizio basato su `converti`, un'altra
-possibilità é quella di indicare le opzioni in un file JSON. Per
-esempio così:
+Se invece state sviluppando un servizio basato su `converti`, è possibile indicare le opzioni in un file JSON. 
```

Comment:
Se invece state sviluppando un servizio basato su `converti`, potete indicare le opzioni in un file JSON in questo modo: